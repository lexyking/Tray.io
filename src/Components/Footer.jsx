import React from 'react'

function Footer() {
  return (
    <footer>
      <p>Tray.io Technical interview, September 2019</p>
    </footer>
  )
}

export default Footer
