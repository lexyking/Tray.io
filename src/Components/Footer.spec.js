import React from 'react';
import { create } from 'react-test-renderer';
import Footer from './Footer';

// Because the footer is an invariable component in this applicationCache, we can test the Snapshot
// and compare it to the previous one

describe('Footer component', () => {
  test('Matches the snapshot', () => {
    const footer = create(<Footer />);
    expect(footer.toJSON()).toMatchSnapshot();
  })
})