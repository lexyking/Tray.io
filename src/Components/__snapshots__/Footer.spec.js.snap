// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Matches the snapshot 1`] = `
<footer
  className="footer"
>
  <p>
    Tray.io Technical interview, September 2019
  </p>
</footer>
`;
